CHANGELOG.md

```diff
@@ -26,6 +26,8 @@
     * Fix #1348: support `v1beta1` version for `ExecCredentials`
 
     * Fix #1354: suppress log warnings that `CustomResourceDefinition`s are still in beta
+    
+    * Fix #1362: store exceptions thrown in port forwarder websocket
```

Comment:
4.1.2 was released a few days ago, Could you please add this to a new section `4.1-SNAPSHOT`?

Reply:
Thanks. Done